src/main/java/org/jenkinsci/plugins/ewm/DiskInfoProvider.java

```diff
@@ -1,45 +1,52 @@
 package org.jenkinsci.plugins.ewm;
 
+import hudson.DescriptorExtensionList;
 import hudson.ExtensionList;
 import hudson.ExtensionPoint;
 import hudson.model.AbstractDescribableImpl;
 import jenkins.model.Jenkins;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 
-import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
 
 /**
+ * Contains additional information about a {@link org.jenkinsci.plugins.ewm.definitions.Disk} entry.
+ *
  * @author Alexandru Somai
  */
 @Restricted(NoExternalUse.class)
 public abstract class DiskInfoProvider extends AbstractDescribableImpl<DiskInfoProvider> implements ExtensionPoint {
 
-    @CheckForNull
-    private final Double writeSpeed;
+    private final double readSpeed;
+    private final double writeSpeed;
```

Comment:
I'm not sure Double is a good idea. Maybe int for KB/s? I doubt we need higher accuracy